pytest/tests/sanity/validator_switch_key_quick.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
```

Comment:
nit: It would be a bit nicer to use the new recommended way of writing pytests - by using the unittests framework. It has all the benefits of using a proper test framework e.g. better assertions and error messages. 
please see simple.py or https://github.com/near/nearcore/tree/master/pytest#creating-new-tests for reference